pipelines/getCategory_v1.json

```diff
@@ -0,0 +1,57 @@
+
+{
+  "version": "1",
+  "pipeline": {
+    "id": "getCategory_v1",
+    "public": true,
+    "input": [
+      {"key": "categoryId", "id": "1"},
+      {"key": "includeChildren", "id": "2", "optional": true},
+      {"key": "childrenSort", "id": "3", "optional": true}
+    ],
+    "output": [
+      {"key": "id", "id": "1000"},
+      {"key": "name", "id": "1001"},
+      {"key": "productCount", "id": "1002"},
+      {"key": "imageUrl", "id": "1003"},
+      {"key": "parent", "id": "1004"},
+      {"key": "children", "id": "1005", "optional": true},
+      {"key": "childrenCount", "id": "1006"}
+    ],
+    "steps": [
+      {
+        "type": "extension",
+        "id": "@shopgate/shopify-catalog",
+        "path": "@shopgate/shopify-catalog/lib/getCategory.js",
+        "input": [{"key": "categoryId", "id": "1"}],
+        "output": [
+          {"key": "id", "id": "1000"},
+          {"key": "name", "id": "1001"},
+          {"key": "productCount", "id": "1002"},
+          {"key": "imageUrl", "id": "1003"},
+          {"key": "parent", "id": "1004"},
+          {"key": "childrenCount", "id": "1006"}
+        ]
+      }, {
+        "type": "conditional",
```

Comment:
Do we actually need this conditional step?

Reply:
when the frontend asks for them we just make a call to our already implemented pipeline and I guess the frontend expects to have (at least an empty array of children) when the includeChildren parameter was sent.

So I would say yes we probably need it